org.widelands.widelands.yaml

```diff
@@ -0,0 +1,28 @@
+app-id: org.widelands.widelands
+runtime: org.freedesktop.Platform
+runtime-version: '1.6'
```

Comment:
Move it to 18.08, [the fix](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/merge_requests/716) is going to be merged soonish. We don't accept 1.6-based apps anymore.

Reply:
The fix has been released.

Reply:
@barthalion 

> Move it to 18.08

Done.